src/aap_eda/wsapi/messages.py

```diff
@@ -38,14 +38,14 @@ class ActionMessage(Message):
     status: Optional[str] = ""
     url: Optional[str] = ""
     rule_run_at: Optional[str] = ""
-    playbook_name: Optional[str]
-    job_template_name: Optional[str]
-    organization: Optional[str]
-    job_id: Optional[str]
-    rc: Optional[int]
-    delay: Optional[float]
-    message: Optional[str]
-    kind: Optional[str]
+    playbook_name: Optional[str] = ""
+    job_template_name: Optional[str] = ""
+    organization: Optional[str] = ""
+    job_id: Optional[str] = ""
+    rc: Optional[int] = 0
+    delay: Optional[float] = 0.0
+    message: Optional[str] = ""
+    kind: Optional[str] = ""
```

Comment:
Why do we need empty default values when the type is already optional?

Reply:
They will be required when pydantic package upgrades

Reply:
Removed the changes required by pydantic 2.5.